### ISSUE TITLE
pgvector with postgresql / alloyDB and AlloyDB OMNI

### ISSUE THREAD

**Issue by pauldintel** [zilliztech/VectorDBBench#273]
With significant performance boost offered by alloyDB postgres Pgvector solution and  Zilliz showing open source pgvector worst performance I would like to see if team can publish and measure the benchmark with alloyDB and show competitive results please. 
thanks

**Comment by alwayslove2013** [comment]
Feel free to contribute the AlloyDB client for performance test and share the performance with us~

**Comment by pauldintel** [comment]
> Feel free to contribute the AlloyDB client for test and share the performance with us~

Sorry if the ask is misunderstood. I am not asking about contribution . Since Zilliz publish the benchmark using VectorDB and pgvector is the last item in the list from performance standpoint . The question is you guys may be missing the AlloyDB pgvector part of the story (which Google published as 4x faster on OLTP and 100x faster on analytics) . Does Zilliz having any future aspirations to add alloydb and alloydb omni in zilliz cloud benchmarking results ??? 

![image](https://github.com/zilliztech/VectorDBBench/assets/104393458/43adf7d6-4c83-43d2-a063-fb2cc6f3d438)

**Comment by alwayslove2013** [comment]
@pauldintel 
Currently, we lack sufficient resources to undertake all requested features and enhancements. Therefore, we warmly invite other community members to contribute code or share their results. We truly appreciate your understanding and patience.

**Comment by xiaofan-luan** [comment]
@alwayslove2013 
is there a pgvector support? if there is , then anyone could simply run the test by themselves and we can add the result to leaderboard

**Comment by alwayslove2013** [comment]
@alwayslove2013 Yes, vdbbench already supports `pgvector` and `pgvectorrs`, and we've posted test results of `pgvector` as well.

**Comment by alwayslove2013** [comment]
In Milestone 1.0.0, we will support testing AlloyDB.

**Comment by pauldintel** [comment]
And perhaps JVector from Cassadra / Datastax too ?

**Comment by xiaofan-luan** [comment]
> And perhaps JVector from Cassandra / Datastax too ?

We are looking for some volunteers.
